src/test/java/org/embulk/filter/row/TestLongCondition.java

```diff
@@ -83,7 +83,70 @@ public void testLessEqual()
     @Test
     public void testNot()
     {
-        LongCondition condition = new LongCondition("<=", new Long(11), true);
+        LongCondition condition = new LongCondition("<=", new Double(11), true);
+        assertTrue(condition.compare(null));
+        assertTrue(condition.compare(new Long(12)));
+        assertFalse(condition.compare(new Long(11)));
+    }
+
+    @Test
+    public void testDoubleOperatorEquals()
+    {
+        LongCondition condition = new LongCondition("==", new Double(10.0), false);
```

Comment:
This is apparently same with `testEquals()` since `new Double(10)` and `new Double(10.0)` are same.
`new Double(10.5)` would be nice to test (which can have no assertTrue case)